docs/_documentations/overview.md

```diff
@@ -1,29 +1,83 @@
+duration: 2 minute
 permalink: Overview
 type: document
 parent: root
 ---
 
-# Overview
+# What is Codewind?
 
-You can use codewind to develop and debug your containerized projects from within an IDE.
+Building and Developing a cloud native, containerized application can be challenging. 
 
-Write code, track application and build statuses, view project logs, and run your application. Codewind supports development of Appsody, Go, Java Lagom, Microprofile/Java EE, Node.js, Python, Spring, and Swift containerized projects. In addition, the tools support easily debugging Microprofile/Java EE, Spring, and Node.js applications.
+Firstly, you need to create your own application stack for a containerized microservice that also fits your preferred language and project type.
 
+Then, if you are a software developer, you need to perform many actions as you develop, build, run, and test your code. You need to build images, assess build status, create containers, open application end points, debug, analyse the different logs, assess performance metrics, and re-build the containers with each code change.
 
-# Features
+Or, if you are responsible for defining standards for application and runtime environments such as framework and software levels, you need to implement and maintain your standards across the whole development team, ensuring consistency and reliability. 
+
+Built from the ground up, Codewind, an [open source project](https://github.com/eclipse/codewind), helps you to achieve all of the above by extending your Integrated Development Environment(IDE). Container development flow feels like a traditional application flow and [project templates](mdt-vsc-workingwithtemplates.html) can be used across your development teams effortlessly. Support for rapid iteration of your code is achieved using features including automatic updating of containers with code changes, pre-supplied container-based [project templates](mdt-vsc-workingwithtemplates.html), and a user interface that exposes metrics, logs, and load testing at the click of a button. Your own standardised [project templates](mdt-vsc-workingwithtemplates.html) can be accessed by your developers using their IDE of choice.
+
+Different development flows are supported, so you can choose whether you want to use a desktop-based or cloud-based IDE, and whether you want to build and run your containers on your desktop or in the Cloud.
+
+Codewind can be used to build and run Docker-containerized applications on [Kubernetes](https://kubernetes.io/) container orchestration applications, including [OpenShift](https://www.openshift.com/), without the need for in-depth knowledge of container technology.
+
+### How does Codewind help me develop containerized applications?
+Codewind improves your inner loop experience, enabling you to create a microservice quickly, rapidly iterate on changes, and make improvements to performance thanks to the following features:
```

Comment:
From the word usage page: Avoid stating that inanimate objects grant abilities to people, as in "the product enables you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can...".

I think it might be easiest to say something like, "...inner loop experience. You can create a..."